cloud_testing/kotlin_poc/flank.ios.yml

```diff
@@ -1,10 +1,11 @@
-xctestrunZip: ./src/test/kotlin/ftl/fixtures/tmp/EarlGreyExample.zip
-xctestrunFile: ./src/test/kotlin/ftl/fixtures/tmp/EarlGreyExampleSwiftTests_iphoneos11.2-arm64.xctestrun
-rootGcsBucket: earlgrey-swift
-disablePerformanceMetrics: false
-disableVideoRecording: false
-testTimeoutMinutes: 30
-testRuns: 1
-testShards: 2
-limitBreak: false
-waitForResults: true
+flank:
+  test: ./src/test/kotlin/ftl/fixtures/tmp/EarlGreyExample.zip
+  xctestrun-file: ./src/test/kotlin/ftl/fixtures/tmp/EarlGreyExampleSwiftTests_iphoneos11.2-arm64.xctestrun
+  results-bucket: earlgrey-swift
+  performance-metrics: false
+  record-video: true
+  timeout: 30
```

Comment:
timeout logic requires additional parsing:

```
--timeout 1h is 1 hour
--timeout 5m is 5 minutes
--timeout 200s is 200 seconds
--timeout 100 is 100 seconds
```
https://cloud.google.com/sdk/gcloud/reference/firebase/test/android/run

The current change takes us from a 60m timeout to a 60 second timeout.